src/DssSpell.t.sol

```diff
@@ -366,36 +262,24 @@ contract DssSpellTest is DssSpellTestBase {
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // RWA015
-        _checkChainlogKey("RWA015_A_JAR");
-        _checkChainlogKey("RWA015");
-        _checkChainlogKey("MCD_JOIN_RWA015_A");
-        _checkChainlogKey("RWA015_A_URN");
-        _checkChainlogKey("RWA015_A_INPUT_CONDUIT_URN");
-        _checkChainlogKey("RWA015_A_INPUT_CONDUIT_JAR");
-        _checkChainlogKey("RWA015_A_OUTPUT_CONDUIT");
-        _checkChainlogKey("PIP_RWA015");
-
         _checkChainlogVersion("1.14.13");
     }
 
-    function testNewIlkRegistryValues() public { // make private to disable
+    function testNewIlkRegistryValues() private { // make private to disable
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
         // Insert new ilk registry values tests here
-        // RWA015
-        (, address pipRwa015,,) = oracle.ilks("RWA015-A");
-
-        assertEq(reg.pos("RWA015-A"),    63);
-        assertEq(reg.join("RWA015-A"),   addr.addr("MCD_JOIN_RWA015_A"));
-        assertEq(reg.gem("RWA015-A"),    addr.addr("RWA015"));
-        assertEq(reg.dec("RWA015-A"),    GemAbstract(addr.addr("RWA015")).decimals());
-        assertEq(reg.class("RWA015-A"),  3);
-        assertEq(reg.pip("RWA015-A"),    pipRwa015);
-        assertEq(reg.name("RWA015-A"),   "RWA015-A: BlockTower Andromeda");
-        assertEq(reg.symbol("RWA015-A"), GemAbstract(addr.addr("RWA015")).symbol());
+        _checkIlkIntegration(
+             "TOKEN-X",
+             GemJoinAbstract(addr.addr("MCD_JOIN_TOKEN_X")),
+             ClipAbstract(addr.addr("MCD_CLIP_TOKEN_X")),
+             addr.addr("PIP_TOKEN"),
+             true,
+             true,
+             false
+        );
```

Comment:
Nit: There is a newline here between ); and } that should be removed

Reply:
What new line?
![image](https://github.com/makerdao/spells-mainnet/assets/112016538/eafdc697-af79-4a94-b40b-c457ce5a624a)

Reply:
My mistake, it was on my local only and affected the diff. Resolving nit